lib/client.js

```diff
@@ -232,6 +232,28 @@ var Client = function(options) {
   this._context = options.context || null;
   this.ready = false;
 
+  var WHITELISTED_ORIGINS = [
+    /.*zendesk\.com.*/gi,
+    /.*zd-staging\.com.*/gi,
+    /.*zd-dev\.com.*/gi,
+    /.*zd-master\.com.*/gi,
+    /.*zendesk-staging\.com.*/gi,
+    /.*dashboard\.zopim\.com.*/gi,
+    /.*dashboard\.zopim\.org.*/gi
+  ];
+
+  var i;
```

Comment:
Couldn't just declare the iterator var in the `for` statement? Seems unnecessary to pull it out.  If you _were_ going to do that, I'd expect you might save the length of the Array in the constant into another var, like `len = WHITELISTED_ORIGINS.length`.